wimm.Secundatives.UnitTests/Extensions/ResultMapping_Test.cs

```diff
@@ -181,6 +180,24 @@ public void MapOr_IsError_ReturnsResultOfErrorFunc()
             Assert.Equal(OtherError.OtherSadness.ToString(), res);
         }
 
+        [Fact]
+        public async Task MapOrTask_IsSuccess_ReturnsResultOfFunc()
+        {
+            var underTest = Task.FromResult(ConstructWith(1));
+            var res = await underTest.MapOr(_ => "deet", _ => OtherError.OtherSadness.ToString());
+
+            Assert.Equal("deet", res);
+        }
+
+        [Fact]
+        public async Task MapOrTask_IsError_ReturnsResultsOfErrorFunc()
+        {
+            var underTest = Task.FromResult(new Result<string, TestError>(TestError.Sadness));
+            var res = await underTest.MapOr(_ => "deet", _ => OtherError.OtherSadness.ToString());
```

Comment:
Should there be a value-task / error-value version? I suppose its easy to make a determined error-value into an error-task so low value but also easy

Reply:
Yeah I think it is a good idea. If you're good with backlogging it I'm just not feeling the effort atm. 🙃